hedyorg/hedy
# [CONTENT] Update babel to 2.14

Fixes #4761 

Sadly Tagalog/ᜆᜄᜎᜓᜄ᜔ (ISO: tl) remains unsupported for now, so I left that override in place, meaning that all babel-translated content (like UI elements) will remain in English for now. 

Our own yaml content like adventures and keywords can be shown properly.

**How to test**

Navigate to the site in Setswana or Papiamentu and observe all still works: http://127.0.0.1:8080/hedy?language=tn#default

Also observe that the one word I translated (Home) is now shown:

<img width="1461" alt="image" src="https://github.com/hedyorg/hedy/assets/1003685/44048b37-c631-4367-a80b-bd9e78d8b8fd">
